lib/signaling/v2/peerconnection.js

```diff
@@ -268,7 +276,16 @@ class PeerConnectionV2 extends StateMachine {
       }
     });
 
-    encodingParameters.on('changed', oncePerTick(this.offer.bind(this)));
+    encodingParameters.on('changed', oncePerTick(() => {
+      if (this._isRTCRtpSenderParamsSupported) {
```

Comment:
@makarandp0 I don't think we should, because there would be no way to know if we have to trigger the offer <--> answer process for cases where the API is not supported.